charts/karpenter/values.yaml

```diff
@@ -49,6 +49,7 @@ podAnnotations: {}
 podDisruptionBudget:
   name: karpenter
   maxUnavailable: 1
+  unhealthyPodEvictionPolicy:
```

Comment:
I suspect that we should set a default policy in our values.yaml if we are going to support this configuration parameter